docs/mosip-installation-without-public-dns.md

```diff
@@ -46,48 +53,90 @@
+  ```
+  output:
+  ```
+    NAME                      READY   STATUS    RESTARTS   AGE
+    coredns-cd565b844-b75zz   1/1     Running   0          7d19h
+    coredns-cd565b844-m8lg5   1/1     Running   0          7d19h
+  ```
+* Update the IP address and domain name in the below DNS hosts template and add it in the coredns configmap `Corefile` key in the `kube-system` namespace.
   ```
    hosts {
-     <PUBLIC_IP>    api.sandbox.xyz.net resident.sandbox.xyz.net idp.sandbox.xyz.net prereg.sandbox.xyz.net
-     <INTERNAL_IP>  sandbox.xyz.net api-internal.sandbox.xyz.net activemq.sandbox.xyz.net kibana.sandbox.xyz.net regclient.sandbox.xyz.net admin.sandbox.xyz.net minio.sandbox.xyz.net iam.sandbox.xyz.net kafka.sandbox.xyz.net postgres.sandbox.xyz.net pmp.sandbox.xyz.net onboarder.sandbox.xyz.net smtp.sandbox.xyz.net
+     <PUBLIC_IP>    api.sandbox.xyz.net resident.sandbox.xyz.net esignet.sandbox.xyz.net prereg.sandbox.xyz.net healthservices.sandbox.xyz.net
+     <INTERNAL_IP>  sandbox.xyz.net api-internal.sandbox.xyz.net activemq.sandbox.xyz.net kibana.sandbox.xyz.net regclient.sandbox.xyz.net admin.sandbox.xyz.net minio.sandbox.xyz.net iam.sandbox.xyz.net kafka.sandbox.xyz.net postgres.sandbox.xyz.net pmp.sandbox.xyz.net onboarder.sandbox.xyz.net smtp.sandbox.xyz.net compliance.sandbox.xyz.net
      fallthrough
    }
   ```
+* Update coredns configmap via below command.
+  ```
+  kubectl -n kube-system edit cm coredns
+  ```
+  example:
   ![mosip-without-dns-1.png](_images/mosip-without-dns-1.png)
+* Check whether the DNS changes are correctly update in coredns configmap.
+  ```
+  kubectl -n kube-system get cm coredns -o yaml
+  ```
 * Restart the `coredns` pod in the `kube-system` namespace.
+  ```
+  kubectl -n kube-system rollout restart deploy coredns coredns-autoscaler
+  ```
+* Check status of coredns restart.
+  ```
+  kubectl -n kube-system rollout status deploy coredns
+  kubectl -n kube-system rollout status coredns-autoscaler
+  ```
 
-## Update config properties
-* Update the below variable in `admin-default.properties`, `resident-default.properties`, and `partner-default.properties`.
+## Update config properties below starting deployment
+* Add/Update the below property in `application-default.properties` and comment on the below property in the `*-default.properties` file in the config repo.
   ```
   mosip.iam.certs_endpoint=http://${keycloak.external.host}/auth/realms/mosip/protocol/openid-connect/certs
   ```
-* Restart admin, resident, and partner services.
+* Add/Update the below property in the `esignet-default.properties` file in the config repo.
+  ```
+  spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://${keycloak.external.host}/auth/realms/mosip/protocol/openid-connect/certs
+  ```
 
-## Local setup
+## Steps to follow while deployment
+* While install few modules, installation script prompts if you have public domain and valid ssl certificates on the server.
```

Comment:
While installing a few modules, installation script prompts to check if you have public domain and valid SSL certificates on the server.